### PR TITLE
Blood Magic Content Update

### PR DESCRIPTION
**Pull request by MuteTiefling**
Creating the groundwork. No JEI integration, so I'm making entries in the patchouli book to show the additional recipes.

Recipes to come and integration into the lang system for the patchouli entries as well.

### CODE REVIEW

File: config/ars_nouveau/chaining.toml [hunk 1]
```diff
@@ -1,28 +1,41 @@
...
-#Bonus to maximum entities per augment
-#Range: > 1
-bonus_entities = 16
-#Base search distance around each target entity
-#Range: 0.0 ~ 1.7976931348623157E308
-base_entity_distance = 8.0
-#Bonus search distance around each target entity per augment
-#Range: 0.0 ~ 1.7976931348623157E308
-bonus_entity_distance = 4.0
+
```
Comment (NielsPilgaard) [inline config/ars_nouveau/chaining.toml:1]: I'm guessing this was added by mistake :D

**Review by NielsPilgaard — COMMENTED** [review]
Very cool so far, nice work with Patchouli

File: config/ars_nouveau/chaining.toml (hunk 1 above)
Comment (MuteTiefling) [reply config/ars_nouveau/chaining.toml:1]: ugh, yes those things sneak in constantly

**Comment by MuteTiefling** [comment]
Just a little sneak peak for those watching but not reading the code:

**Comment by MuteTiefling** [comment]
Updates recipes for Teleposers in Expert, bringing them forwards a bit compared to standard progression
image

image

The 3x3 and 5x5 versions remain more deeply gated.

Also introduces some quests for new brewing mechanics, the teleposer, a mention on meteors, and the new will growth mechanics.

**Comment by MuteTiefling** [comment]
Attempts to convert this to lang entries didn't work. Gonna leave them as hard coded for now. Might revisit this later.

**Comment by MuteTiefling** [comment]
Special thanks to Howester and GenerikB for the design of many of the shells too :D

File: kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/bloodmagic/meteor.js [hunk 2]
```diff
@@ -0,0 +1,648 @@
...
+            explosion: 0.0,
+            layers: [],
+            id: patchouli_removal_by_id
+        });
+    });
+
+    recipes.forEach((recipe) => {
+        recipe.type = 'bloodmagic:meteor';
+        recipe.input = Ingredient.of(recipe.input).toJson();
+        //set cost to 0 for testing. Remove before merge
```
Comment (NielsPilgaard) [inline kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/bloodmagic/meteor.js:644]: Remember this :D

**Review by NielsPilgaard — APPROVED** [review]
Looks very good, seems you left yourself a TODO though 😄

**Comment by MuteTiefling** [comment]
Lol, I knew I would

**Comment by MuteTiefling** [comment]
There, should be good. Did from mobile, so who knows!

**Pull request merged**